### DISCUSSION TITLE
Strano comportamento di connettività dei nodi Zigbee

### DISCUSSION THREAD

**Discussion by paolomorabito** [indomus/forum#1650] [ZigBee, Z-Wave, Thread, Matter]
Ho una rete zigbee con circa 30 dispositivi (di tipo Aqara) e 3 routers (Ikea), tramite Conbee II (con ultimo firmware) su Hassio. Tutto funziona correttamente ma, di tanto in tanto, i dispositivi cominciano a funzionare male.  A volte si collegano, a volte no. A volte rispondono correttamente, altre volte no. Se apro (da Hassio) Phoscon i nodi sembrano apparentemente collegati (ma in realtà non sempre rispondono). Se apro Deconz, invece, vedo che tanti nodi non sono più collegati. 

Quando ciò accade, l'unica cosa che rimette a posto la situazione è il pairing (nuoavmente) di tutti i dispositivi. Poi tutto funziona, anche per diverse settimane, fin quando accade qualcosa (non so cosa) e tutto torna punto e accapo. 

Non sono le batterie dei dispositivi (già più volte sostituite). Non è la distanza (i dispositivi sono molto vicini ai routers e questi ultimi sono sempre collegati al coordinator)

Qualcuno ha mai notato un comportamento del genere? E come ha risolto?
Esiste un modo più veloce per riacquisire la connettività di tutti i dispositivi?

Grazie

**Comment by nromagna** [comment]
Ciao ho ho una 10ina di dispositivi ZB, collegati tra loro con router sonoff e con firmware originale. Mi sono accorto che se da tasto fisico (ZB) o da app accendo spengo velocemente una lampadina si "impallano tutti" e diventano tutti non raggiungibili da HA.
L'unico modo, per me, è quello di riavviare il router ZB ed attendere qualche secondo. Ritoranano tutti funzionanti.

**Comment by paolomorabito** [comment]
Ho provato a riavviare (nel mio caso devo staccare la chiavetta Conbee 2 da usb). Il problema persiste

**Comment by indomus** [comment]
> 

Premesso che è un consiglio stupido - la cosa migliore sarebbe indagare - ma conoscendo Deconz, attualmente il consiglio migliore è migrare a ZigBee2MQTT. Con ConBee II funziona da dio, evitandoti molti problemi che invece con Deconz hai.

**Comment by paolomorabito** [comment]
Io penso sia più una questione di hardware. Perché se i dispositivi si dissociano dal coordinator, allora vuol dire che c'è qualcosa che non va a livello hardware. Se usassi Zigbee2mqtt mi ritroverei nella stessa situazione.

Ovviamente è una supposizione.

**Reply by snakuzzo** [reply]
Il fatto è che tu PENSI che abbiano perso l'associazione...non ne sei certo...non hai una prova evidente di questo.

**Reply by paolomorabito** [reply]
Suggerimenti?

**Reply by snakuzzo** [reply]
Ti è già stato dato il suggerimento. [QUI](https://github.com/indomus/forum/discussions/1650#discussioncomment-2598769)

**Reply by snakuzzo** [reply]
Butta via Deconz e prova [ZHA](https://www.home-assistant.io/integrations/zha/) o [Zigbee2mqtt](https://indomus.it/componenti/zigbee2mqtt-bridge-gateway-software-zigbee/)
